src/sentry/templates/sentry/error-page-embed.js

```diff
@@ -87,25 +91,47 @@
       }
     }
 
-    var linkTags = this.element.getElementsByTagName('a');
-
-    var onclickHandler = function(e) {
+    var cancelHandler = function(e) {
       e.preventDefault();
       self.close();
     };
-
-    for (i = 0; i < linkTags.length; i++) {
-      if (linkTags[i].className === 'close') {
-        linkTags[i].onclick = onclickHandler;
-      }
-    }
+    buttons[1].onclick = cancelHandler;
 
     this._formMap = {};
     var node;
     for (i = 0; i < this._form.elements.length; i++) {
       node = this._form.elements[i];
       this._formMap[node.name] = node.parentNode;
     }
+
+    if (document.querySelectorAll) {
+      var focusableElements = this.element.querySelectorAll(focusable);
+      var firstFocus = focusableElements[0];
+      var lastFocus = focusableElements[focusableElements.length - 1];
+      // Trap focus to improve UX and help screenreaders.
+      handleFocus = function (event) {
+        var isTab = event.key === 'Tab' || event.keyCode === 9;
+        if (!isTab) {
+          return;
+        }
+        if (event.shiftKey) {
+          if (document.activeElement === firstFocus) {
+            lastFocus.focus();
+            event.preventDefault();
+          }
+        } else {
+          if (document.activeElement === lastFocus) {
+            firstFocus.focus();
+            event.preventDefault();
+          }
+        }
+      }
+      document.addEventListener('keydown', handleFocus);
+
+      setTimeout(function () {
+        firstFocus.focus();
+      }, 1)
+    }
```

Comment:
Clever way to trap focus, feels like the browser should have something built in for this.

Reply:
I wish there was a built-in way.